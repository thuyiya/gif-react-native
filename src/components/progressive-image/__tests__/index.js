/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { ProgressiveImage } from "_components";

describe("ProgressiveImage", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <ProgressiveImage
        thumbnailSource={{ uri: "" }}
        source={{ uri: "" }}
        resizeMode={"cover"}
      />
    );
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
