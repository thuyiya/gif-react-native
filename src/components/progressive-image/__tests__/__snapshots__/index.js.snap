// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressiveImage renders correctly react-test-renderer 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#e1e4e8",
    }
  }
>
  <ForwardRef(AnimatedComponentWrapper)
    blurRadius={1}
    onLoad={[Function]}
    resizeMode="cover"
    source={
      Object {
        "uri": "",
      }
    }
    style={
      Array [
        undefined,
      ]
    }
  />
  <ForwardRef(AnimatedComponentWrapper)
    onLoad={[Function]}
    resizeMode="cover"
    source={
      Object {
        "uri": "",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "opacity": 0,
        },
        undefined,
      ]
    }
  />
</View>
`;
